### DISCUSSION TITLE
古墓丽影崛起无法运行

### DISCUSSION THREAD

**Discussion by Wintere107** [Gamuxorg/bbs#219] [游戏或网站运行反馈]
操作系统：deepinv20

bug描述：无法使用Vulkan运行（无该选项），opengl下运行时，显示系统不支持3d api

说明：已装好vulkan驱动，可以使用dxvk，vulkaninfo输出正常。lutris及steam，proton均可正常使用。opengl下能运行gamux的古墓丽影9。推测打包文件有问题，解压后运行问题依旧

**Comment by shouhuanxiaoji** [comment]
经过测试，可能是deepin20的vulkan驱动有问题，导致无法正确识别显卡型号。

**Comment by Wintere107** [comment]
> 经过测试，可能是deepin20的vulkan驱动有问题，导致无法正确识别显卡型号。

后续用过arch测试，问题相同，推测为文件问题，需要更新

**Comment by shouhuanxiaoji** [comment]
在manjaro也测试可运行，您的硬件是什么

**Comment by Wintere107** [comment]
收到邮件
